dir_class.py

```diff
@@ -31,19 +32,23 @@ def get_control_string(self):
     def get_music_files_count(self):
         return len(self.music_files)
 
-    def move(self, destination_subdir, destination_dir):
-        for x, y, z in os.walk(current_dir):
-            destination_pth = destination_dir + os.sep + destination_subdir
+    def move(self, destination_dir, destination_subdir):
+        for x, y, z in os.walk(self.base_dir):
+            destination_pth = destination_dir + os.sep
             os.makedirs(destination_pth, exist_ok=True)
```

Comment:
Po pierwsze - powinno być `destination_pth = destination_dir + os.sep + destination_subdir`.

Ponieważ dla każdej iteracji tej pętli `destination_pth` będzie dokładnie takie samo, to obie powyższe linijki możesz wykonać jeszcze przed wejściem w pętle. Po co obliczać ścieżkę za każdym razem, skoro się nie zmienia.
To Ci rozwiąże też problem z tym, że katalog już istnieje.

Reply:
Nie rozwiązało.

Reply:
Rozwiązało, tylko teraz masz ten sam problem kawałek dalej.

Powinno być 
```
            for f in z:
                source_pth = x + os.sep + f
                shutil.move(source_pth, destination_pth)
```